### DISCUSSION TITLE
Master/detail (parent-child) form?

### DISCUSSION THREAD

**Discussion by catamaican** [formkit/formkit#1261] [General]
Is it possible to create a master/detail (parent/child) forms? (e.g. class/students or order/items-on-order). If yes, could you, please, provide a demo?

Thank you!

**Comment by fenilli** [comment]
You should not by HTML specs, have forms within forms tho, what would be a use case for this?

**Comment by catamaican** [comment]
Well, it could be any form of master/detail, for example for one order that
the user selects, I would like to display to the user all the items in that
order, with the possibility of CRUD for those items

It doesn't have to be HTML form in another one, since the order details
could be a dynamic table in the same form, I think

**Comment by fenilli** [comment]
The thing is, for most cases in this, both the parent and child are separated calls to the database correct? its not the parent job to get those properties in a crud to change the children, so for these cases you would have a form for the parent elements, and a form for the children elements and they don't know anything about each other, other than getting maybe the ID and rendering the children.

In that case you can one, have it separate in two forms, or have a form with the parent, have children as group `:ignore=true` so it isn't part of the parent form, and now you can add the same functionallity of submiting inside that group that a form has and each one sends to the correct crud.

Can you explain a bit more on what exactly you need with those in mind, and where FormKit would help with that?
